packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
+    auth_mode: OAUTH2
+    authorization_url: https://acuityscheduling.com/oauth2/authorize
+    token_url: https://acuityscheduling.com/oauth2/token
+    connection_configuration:
+        - userId
+        - apiKey
+    post_connection_script: acuityPostConnection
```

Comment:
- You may want to consider omitting these two, as a connection hook isn't present for Acuity Scheduling
- Additionally, the field in `connection_configuration` is parsed from the post-connection script, so it might be best to omit this as well.

Reply:
🫡